GliaWidgets/Lib/Presenter/GliaPresenter.swift

```diff
@@ -0,0 +1,34 @@
+import UIKit
+internal final class GliaPresenter {
+    private var window: UIWindow? {
+        if #available(iOS 13, *) {
+            return UIApplication.shared.windows.first { $0.isKeyWindow }
+        } else {
+            return UIApplication.shared.keyWindow
+        }
+    }
+
+    internal var root: UIViewController {
+        guard let window = window else {
+            fatalError("Could not find key UIWindow to present on")
+        }
+
+        guard var presenter = window.rootViewController else {
+            fatalError("Could not find UIViewController to present on")
+        }
+
+        while let presentedViewController = presenter.presentedViewController {
+            presenter = presentedViewController
+        }
+
+        return presenter
+    }
+
+    internal func present(_ viewController: UIViewController, animated: Bool, completion: (() -> Void)? = nil) {
+        root.present(viewController, animated: animated, completion: completion)
+    }
+
+    internal func dismiss(_ viewController: UIViewController, animated: Bool, completion: (() -> Void)?) {
+        viewController.presentingViewController?.dismiss(animated: animated, completion: completion)
```

Comment:
Do you expect of changing logic here?
Looks like before this PR the dismiss method called for `topMostViewController` ([here](https://github.com/salemove/ios-sdk-widgets/pull/82/files#diff-c55b54fda28a347f2fdd0c742537e17ab4335a1e4f9f4a0ff227c5441e11bef3L261))

Reply:
Yes, this is the actual fix. Previously, topMostViewController was a computed property, so by the time someone calls dismisis on it, the current topMostViewController could be something totally different. So the right approach is to get the vc, that presented Glia, and call dismiss on that.

Reply:
As I understood judging by the 👍 , you agree with the explanation provided. Resolving :)